webapp/channels/src/components/admin_console/system_users/system_users_list_actions/deactivate_member_modal.tsx

```diff
@@ -34,6 +35,8 @@ export default function DeactivateMemberModal({user, onExited, onError}: Props)
         const {error} = await dispatch(updateUserActive(user.id, false));
         if (error) {
             onError(error);
+        } else {
+            onSuccess();
```

Comment:
Where would an error be thrown that we aren't already handling? If it's something `dispatch` throws I think having that unhandled makes sense since we don't expect it.